app/lib/frontend/handlers/landing.dart

```diff
@@ -46,13 +47,34 @@ Future<shelf.Response> indexLandingHandler(shelf.Request request) async {
   }
 
   Future<String> _render() async {
-    final taggedPackages = await topFeaturedPackages(
+    final ffPackages = await topFeaturedPackages(
       requiredTags: [PackageTags.isFlutterFavorite],
+      count: requestContext.isExperimental ? 3 : 6,
+      emptyFallback: true,
     );
-    final topPackages = await topFeaturedPackages();
+    final topPackages =
+        requestContext.isExperimental ? null : await topFeaturedPackages();
+
+    // TODO: implement top rated
+    final mostPopularPackages = requestContext.isExperimental
+        ? await topFeaturedPackages(order: SearchOrder.popularity)
+        : null;
+
+    // TODO: implement top trending
```

Comment:
No need to track this is TODOs, file an issue... if I recall correctly we've agreed to not implement that as part of the design.. maybe not even this Q.